src/components/Modal/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './auto'
-export { Modal, ModalProps, ModalSize } from './Modal'
+export { Modal, ModalProps, ModalSize, ModalScroll } from './Modal'
 export { ModalBackdrop, ModalBackdropProps } from './ModalBackdrop'
 export { ModalBody, ModalBodyProps } from './ModalBody'
 export { ModalContainer, ModalContainerProps } from './ModalContainer'
 export { ModalFooter, ModalFooterProps } from './ModalFooter'
+export { ModalFooterButton } from './ModalFooterButton'
```

Comment:
Maybe the original pattern here in bold is to also export the props. Can you check it?